runtime/src/bank.rs

```diff
@@ -512,6 +517,27 @@ impl Bank {
             txs.len(),
         );
         self.update_transaction_statuses(txs, &executed);
+
+        let mut fees = 0;
```

Comment:
Can you put all of this into a new function? Looks like a lot of edge cases we'd want to test independently of the code around it.

Reply:
That can happen in a follow-up PR

Reply:
👍 will do.